packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -308,6 +310,28 @@ export default {
         return callTree;
       }, []);
 
+      if (this.rootCodeObject) {
+        const eventBranches = this.rootCodeObject.allEvents.map((e) => [
+          e.id,
+          e.linkedEvent.id,
+        ]);
+
+        events = events.filter((e) =>
+          eventBranches.some((branch) => e.id >= branch[0] && e.id <= branch[1])
+        );
+
+        if (
+          this.rootCodeObject.type === 'class' &&
+          this.rootCodeObject.packageObject
+        ) {
+          classMap = new ClassMap([
+            classMap.codeObjects.filter(
+              (co) => co.id !== this.rootCodeObject.packageObject.id
+            ),
+          ]);
+        }
+      }
```

Comment:
Would you mind adding a comment to this block explaining it's purpose, similar to what you described in Slack?
```
It's necessary to remove package wrap in Component Diagram, without this code package will be rendered in collapsed state
```

Reply:
Comment was added https://github.com/applandinc/appmap-js/pull/390/commits/80ad5d783b816686954a494d3fba42f28eebb5af